README.md

```diff
@@ -52,17 +48,35 @@ SNAPSHOT=true DEV=true PLATFORMS=darwin mage package
 The above assumes you are running on OS X. Put the platform in you are running on. This speeds up packaging as it only builds it for your platform. As soon as this is completed (it might take a while for the first time) navigate to `build/distributions` and unpackage the `.tar.gz`. Navigate into the elastic-agent directory and start the Elastic Agent:
 
 ```
-sudo ./elastic-agent -v
+KIBANA_HOST=http://localhost:5601 KIBANA_USERNAME=elastic KIBANA_PASSWORD=changeme ELASTICSEARCH_HOST=http://localhost:9200 ELASTICSEARCH_USERNAME=elastic ELASTICSEARCH_PASSWORD=changeme KIBANA_FLEET_SETUP=1 FLEET_SERVER_ENABLE=1 sudo ./elastic-agent container
 ```
 
-Currently a second command has to be run in parallel to setup fleet-server. Take the enrollment token and policy id you copied from Kibana and replace it in the command below:
+This will start up Elastic Agent with fleet-server and directly enroll it. In addition Fleet is setup.
```

Comment:
```suggestion
This will start up Elastic Agent with fleet-server and directly enroll it. In addition Fleet is setup inside of Kibana.
```

At first read I was not clear at what that meant, until I read it again I believe you mean it will setup Fleet in Kibana.